02-IDE_exercises/solutions/09-project/AddThreeGame.py

```diff
@@ -18,38 +18,37 @@ class AddThreeGame:
     then the game ends in a draw.
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
         """
         intializes 4 private variables to be used throughout the Class
         """
-        self._nums_chosen = []
-        self._current_state = "UNFINISHED"
-        self._player_turn = "first"
-        self._player_score = 0
+        self._nums_chosen:  list[None] = []
```

Comment:
incorrect type hinting for a list, see previous comments. Should also be declared with `List[int]` or `List[float]` if the list is expected to store nums